resources/views/components/game/compare-progress.blade.php

```diff
@@ -0,0 +1,83 @@
+<?php
+use App\Models\User;
+?>
+
+@props([
+    'game' => null,
+    'user' => null,
+    'followedUserCompletion' => null,
+])
+
+<?php
+$baseUrl = str_replace($user->User, '', route('game.compare-unlocks', [$game, $user]));
+?>
+
+<script>
+jQuery(document).ready(function onReady($) {
+  var $searchBoxCompareuser = $('.searchboxgamecompareuser');
+  $searchBoxCompareuser.autocomplete({
+    source: function (request, response) {
+      request.source = 'game-compare';
+      $.post('/request/search.php', request)
+        .done(function (data) {
+          response(data);
+        });
+    },
+    minLength: 2
+  });
+  $searchBoxCompareuser.autocomplete({
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    select: function (event, ui) {
+      return false;
+    },
+  });
+  $searchBoxCompareuser.on('autocompleteselect', function (event, ui) {
+    window.location = "{!! $baseUrl !!}" + ui.item.label;
+    return false;
+  });
+});
+
+function selectSearchBoxUser() {
+  var $searchBoxCompareuser = $('.searchboxgamecompareuser');
+  window.location = "{!! $baseUrl !!}" + $searchBoxCompareuser.val();
+}
+</script>
+
+<div id='gamecompare' class='component'>
```

Comment:
Minor nit: due to how the pages were written previously our html attributes use single quotes. It's common practice to use double quotes for html attributes, for newly written or ported blade views I'd like to aim for this cleanup step if possible (I'll probably do another chore PR just for that at some point).